src/lib/markdown.js

```
@@ -0,0 +1,260 @@
+          raw: '',
+          text: '',
+          tokens: [{
+            type: 'text',
+            raw: '',
+            text: ''
+          }]
+        }]
+      };
+
+      // Check if this is a task item
+      if (token.classes && token.classes.includes('task-list-item')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)